webapp/package.json

```diff
@@ -10,7 +10,7 @@
     },
     "dependencies": {
         "core-js": "3.1.4",
-        "mattermost-redux": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
+        "mattermost-redux": "5.33.0",
```

Comment:
Unrelated change

Reply:
@hanzei made this changes because i wasn't able to deploy the plugin , 
@mickmister should i revert this ?